remo/profiles/api/serializers.py

```diff
@@ -41,6 +41,8 @@ class UserProfileDetailedSerializer(serializers.HyperlinkedModelSerializer):
     groups = GroupSerializer(source='user.groups', many=True)
     functional_areas = FunctionalAreaSerializer(many=True)
     remo_url = serializers.SerializerMethodField()
+    date_joined_program = serializers.DateTimeField(format='%Y-%m-%d')
+    date_left_program = serializers.DateTimeField(format='%Y-%m-%d')
```

Comment:
Yes, django rest framework has changed the way is handling DateFields/DateTimeFields. Basically these are the same db fields with a bit of formatting. 

http://www.django-rest-framework.org/topics/3.0-announcement/#date-and-time-objects-as-iso-8859-1-strings-in-serializer-data